### ISSUE TITLE
Page title (Multi language) field empty after core upgrade to 3.0.165

### ISSUE THREAD

**Issue by jmartsch** [processwire/processwire-issues#1291]
I have a page that uses the template "stelle" which has a Page title (Multi language) field named "title". As soon as I update the core from version 3.0.123 to version 3.0.165, the title field is empty. When I revert back to the old core version, the field is filled out again.

The template "stelle" has "Disable multi-language support for this template?" enabled.

This bug could be related to https://github.com/processwire/processwire-issues/issues/1118.

**Comment by jmartsch** [comment]
It turns out, that the title value is only set in my second language (german) which has the language Id 1012.
A simple database query showed, that the field "data" is empty. As soon as I upgrade the core or change the field type to "Page Title" (without Multi language), the value from the empty data field is taken, and not from data1012.

![image](https://user-images.githubusercontent.com/551551/103290758-9079ac00-49ea-11eb-83ee-6e077fd31673.png)

**Comment by jmartsch** [comment]
It turns out, that the title value is only set in my second language (german) which has the language Id 1012.
A simple database query showed, that the field "data" is empty. As soon as I upgrade the core or change the field type to "Page Title" (without Multi language), the value from the empty data field is taken, and not from data1012.

![image](https://user-images.githubusercontent.com/551551/103290758-9079ac00-49ea-11eb-83ee-6e077fd31673.png)

I just used a SQL query to copy the values from the german language data field "data1012" to the "data" field, to fix the issue:
`UPDATE field_title SET data=data1012 WHERE data = ''`

I think this error is coming from some function that I used earlier and only filling the title field in a specific language. `$uploadpage->title->setLanguageValue($language, $form_fields['title']['value']);`
